test/parallel/test-http-allow-req-after-204-res.js

```diff
@@ -49,7 +49,7 @@ function nextRequest() {
         // throws error:
         nextRequest();
         // works just fine:
-        //process.nextTick(nextRequest);
+        // process.nextTick(nextRequest);
```

Comment:
Please add `TODO: investigate why this does not work fine even though it should.`.